### ISSUE TITLE
NorthStar Service Hall Light

### ISSUE THREAD

**Issue by trazodont** [Skyrat-SS13/Skyrat-TG#22851]
Reporting client version: 514.1589

## Round ID:
9049

## Testmerges:
- [[SEMI-MODULAR] Roundstart delam panic button](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22145)
- [[MODULAR] Adds the option for missing a limb to the prosthetics menu](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22097)
- [[MANUAL MIRROR] Robotic organ and disease improvements](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22631)
- [Hemophagus, The Hemophage Revamp, Part Four: Blood Depravity](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22625)
- [Makes certain organs properly unremovable](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22828)
- [Company Imports: Affordable Handguns Addition and Tweaks](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22493)

## Reproduction:

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/97755577/441615f7-69bc-482f-867d-6c7c3cb71576)

Basically the light switch for the service hall lights is not even "in" the service hall, it's in the locker rooms.
